scripts/token-create/create-token.go

```diff
@@ -0,0 +1,132 @@
+/*
+* Copyright 2021 LimeChain Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+ */
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+
+	"github.com/hashgraph/hedera-sdk-go"
+)
+
+func main() {
```

Comment:
I think it would be good to have 1 more script: `Associate to Token`. Input params  - `privateKey`, `accountID`, `network`, `tokenID`